src/pages/AddGamePage.js

```diff
@@ -1,14 +1,29 @@
 import styled from 'styled-components/macro'
 import RemoveButton from '../components/RemoveButton'
 import Search from '../components/Search'
+import '@fontsource/play'
 
-export default function AddGamePage({ onSubmit, games, onRemove }) {
+import Logo from '../images/logo.png'
+
+export default function AddGamePage({
+  onSubmit,
+  games,
+  onRemove,
+  handleShowDetails,
+}) {
   return (
     <Container>
+      <LogoContainer>
+        <img src={Logo} alt="" />
```

Comment:
please consider using cumulative Image Shift with your pictures :) --> https://www.smashingmagazine.com/2021/06/how-to-fix-cumulative-layout-shift-issues/